jest.config.ts

```diff
@@ -16,6 +16,7 @@ const {
   CI_NODE_INDEX,
   GITHUB_PR_SHA,
   GITHUB_PR_REF,
+  GITHUB_RUN_ID,
```

Comment:
I wonder if it would be helpful to add `GITHUB_RUN_ATTEMPT` as well - this would tell us when people re-run workflows via GH's UI (e.g. if it's > 1, then it is a re-run).